Add Flutter SR documentation on Strip Style (#36194)
DataDog/documentation: layouts/shortcodes/mdoc/en/real_user_monitoring/session_replay/setup_and_configuration.mdoc.md

}
```

> ### Step 5 - Set Symbol Strip Style (iOS Only)

Reviewer: ```suggestion
### Step 5 - Set symbol strip style (iOS only)
```

Author: Actually, this should probably be:
```suggestion
### Step 5 - Set Strip Style (iOS only)
```

As "Strip Style" is the name of the setting as it appears in XCode. Would that be correct?

Reviewer: Yes that works, thanks!

Flutter Session Replay relies on FFI, and some symbols may be stripped when building an iOS Archive or IPA. To prevent this, you

> will need to set the `Strip Style` setting in your XCode project to `Non-Global Symbols`.

Reviewer: ```suggestion
Flutter Session Replay relies on FFI, and some symbols may be stripped when building an iOS Archive or IPA. To prevent this, set the `Strip Style` setting in your XCode project to `Non-Global Symbols`.
```

> This setting is under the `Runner` target, under `Build Settings`, `Deployment`.

Reviewer: ```suggestion
This setting is under the **Runner** target > **Build Settings** > **Deployment**.
```
